commcare_export/commcare_minilinq.py

```diff
@@ -41,7 +41,7 @@ def __call__(self, since, until):
             range_expression['lte'] = until.isoformat()
 
         server_modified_missing = {"missing": {
-            "field": "server_modified_on", "null_value": True, "existence": True}
+            "field": "inserted_at", "null_value": True, "existence": True}
```

Comment:
this complex filter is not necessary for `inserted_at` since all the records should have this data. It was only necessary for the modified on field since that was added without a full reindex so it was missing in older data.

See https://github.com/dimagi/commcare-export/pull/124/files#diff-5186f4dc371d1d524367519e15e4eae5L34

Reply:
thanks, updated in 39c06df

Reply:
I think the whole `FormFilterSinceParams` class can be removed and we can just use the `SimpleSinceParams` class (unless you think there is some ES data that doesn't have the `inserted_at` field.

Reply:
makes sense, updated in ab9a9f4